roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,59 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names
+        if int(master_count) > 1:
+            active = "etcd" in group_names
+
+        return super(EtcdTraffic, cls).is_active(task_vars) and active
+
+    def run(self, tmp, task_vars):
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        deploy_type = get_var(task_vars, "openshift", "common", "service_type")
+        unit = self._get_systemd_unit(master_count, deploy_type)
+
+        log_matchers = [{
+            "start_regexp": r"Starting Etcd Server",
+            "regexp": r"etcd: sync duration of \d+\.\d+s, expected less than 1s",
```

Comment:
The variable part here should be probably more flexible:
The `.` is probably optional, and since this is coming from a Go duration value... it may end with `s`, but also `m` for minutes, etc, right?
https://golang.org/pkg/time/#Duration.String

Suggestion:

```python
r'etcd: sync duration of [^,]+, expected less than 1s'
```

Matches anything that is not a comma, then the comma and the rest of the error message.